doc/source/examining/loading_data.rst

```diff
@@ -271,16 +271,18 @@ will be in code units.
 
 .. _loading-orion-data:
 
-BoxLib Data
+AMReX / BoxLib Data
 -----------
 
-yt has been tested with BoxLib data generated by Orion, Nyx, Maestro, Castro, 
-and WarpX. Currently it is cared for by a combination of Andrew Myers, Chris
-Malone, Matthew Turk, and Mike Zingale.
+AMReX and BoxLib share a frontend (currently named `boxlib`), since
+the file format is identical.  yt has been tested with AMReX/BoxLib
```

Comment:
good point, I'll add "nearly".  I imagine it won't matter much in the near future, since BoxLib use will drop off rapidly as people move to AMReX